### PR TITLE
Rotation matrix output columns aligned.

### PR DESCRIPTION
**Pull request by malibrud**
Hello,

I love this library.  I changed the format of the output rotation matrix so that the columns align.  To do this I added a function to convert to fixed width strings of each matrix element.

Mark Brudnak

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -171,7 +171,10 @@
 function matrixToString(m)
 {
     var r = m.elements;
-    var s = '[ '+toReal(r[0])+', '+toReal(r[4])+', '+toReal(r[8])+';\n'+toReal(r[1])+', '+toReal(r[5])+', '+toReal(r[9])+';\n'+toReal(r[2])+', '+toReal(r[6])+', '+toReal(r[10])+' ]';
+    var s = 
+         '[ ' +pmOneToStr(r[0],7)+', '+pmOneToStr(r[4],7)+', '+pmOneToStr(r[8],7)+';\n'
```
Comment (gaschler) [inline index.html]: Since you only use this function with argument "7", you can drop the argument and make 7 default.

File: index.html [hunk 2]
```diff
@@ -331,6 +334,18 @@
 		return x;
 	}
 }
+function pmOneToStr(x, p){
```
Comment (gaschler) [inline index.html]: pmOneToStr is cryptic, use toOutputString or prettyNumber instead.

File: index.html [hunk 3]
```diff
@@ -331,6 +334,18 @@
...
+	if (!isNaN(parseFloat(x)) && isFinite(parseFloat(x)))
+	{
+		var s = x.toFixed(p);
+                if (x >=0) s = ' ' + s;
```
Comment (gaschler) [inline index.html]: space missing ">= 0"

**Review by gaschler — CHANGES_REQUESTED** [review]
Thank you very much for the contribution.
I have only minor remarks before it can be merged.

File: index.html (hunk 1 above)
Comment (malibrud) [reply index.html]: I hard coded the 7.

File: index.html (hunk 2 above)
Comment (malibrud) [reply index.html]: I changed name to 'toFixedWidth'

File: index.html (hunk 3 above)
Comment (malibrud) [reply index.html]: I added the space.

**Comment by malibrud** [comment]
I made all requested changes.

**Review by gaschler — APPROVED** [review]

**Comment by gaschler** [comment]
Thanks again for your patch, merged it.

**Pull request merged**